### PR TITLE
feat(bucket): Switch nirsoft bucket to ScoopInstaller/Nirsoft

### PR DESCRIPTION
**Pull request by rasa**
Per https://github.com/ScoopInstaller/Scoop/discussions/5293

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I have ensured that I am targeting the `develop` branch.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.
- [x] I have added an entry in the CHANGELOG.

### CODE REVIEW

**Comment by rasa** [comment]
Should we instead create a "nirsoft2" bucket to allow users to "upgrade" to the new bucket when they're ready?

**Comment by rashil2000** [comment]
> @ScoopInstaller/maintainers : Should we instead create a "nirsoft2" bucket to allow users to "upgrade" to the new bucket when they're ready?

I don't think that should be necessary.

However, I would like to stress again on the fact that we need more bucket maintainers, since more and more buckets are getting moved into the ScoopInstaller org. Should we make a community call for Nirsoft bucket, preferably before this PR is actually merged?

**Review by niheaven — APPROVED** [review]

**Pull request merged**
